framework/src/Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic/Themes/Basic/Layouts/Application.cshtml

```diff
@@ -25,11 +25,17 @@
 
         pageTitle = PageLayout.Content.Title + pageTitle;
     }
+
+    var pageLang = ViewBag.Lang == null ? "" : $" lang={ViewBag.Lang}";
+    if (PageLayout.Content.Lang != null)
+    {
+        pageLang = $" lang={PageLayout.Content.Lang}";
+    }
 }
 
 <!DOCTYPE html>
 
-<html lang="en">
+<@($"html{pageLang}")>
```

Comment:
```suggestion
<html @pageLang>
```

It is not good to move fundamental html tag to a dynamic string which may break VS editor (it may not recognize the html opening tag). I suggest the above syntax.

Also, put language code inside " and ". Result will be like:

````xml
<html lang="fr">
````
````

Reply:
Ok, I update it, just want to remove the extra space if lang is empty.

Reply:
Then you can do this:

````xml
<html@pageLang>
````

Reply:
There is much simpler solution, by the time layout is rendered culture is already defined in the CultureInfo so it can be written like this and it will work universally with no other code required. Even if it were null (which shouldn't happen) Razor is smart enough to not render tags that are null. This can be done in all 3 basic theme layouts.

    <html lang="@CultureInfo.CurrentUICulture.Name">

Requires using in each layout file or even better can be specified once in the related _ViewImports.cshtml

    @using System.Globalization